app/screens/home-screen/home-screen.tsx

```diff
@@ -137,6 +150,16 @@ export const HomeScreen: React.FC = () => {
 
   const onMenuClick = (target: Target) => {
```

Comment:
I'm not sure it's best to show this menu on the second time someone would want to receive. that could make sense but maybe having some modal after someone do successfully an action, not before, would be best? at the same time I understand that here it's also a great time to ask for this early.

Reply:
I think it makes sense to place it here because you dont slow down the first interaction with the application but its natural to ask which wallet you want to receive to before you navigate to the receive screen.